Doc/faq/windows.rst

```diff
@@ -170,11 +170,12 @@ offender.
 How do I make an executable from a Python script?
 -------------------------------------------------
 
-See http://cx-freeze.sourceforge.net/ for a distutils extension that allows you
-to create console and GUI executables from Python code.
-`py2exe <http://www.py2exe.org/>`_, the most popular extension for building
-Python 2.x-based executables, does not yet support Python 3 but a version that
-does is in development.
+See http://cx-freeze.sourceforge.net/ and http://www.pyinstaller.org/ for the 
+distutils extensions that allows you to create console and GUI executables 
```

Comment:
PyInstaller isn't a distutils extension (and AFAIK doesn't even have anything to do with distutils).